### DISCUSSION TITLE
Facing Problem After Installing (Install-Package Serenity.Extensions)

### DISCUSSION THREAD

**Discussion by moseur** [serenity-is/Serenity#6462] [Help]
Hello All,

I am newbie with Serenity. Just trying MovieTutorial. After ( Install-Package Serenity.Extensions)  I am facing below listed problems and I am not able to build my project. Can anybody help me.

Please find the error details below:

Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0246	The type or namespace name 'EnvironmentSettings' could not be found (are you missing a using directive or an assembly reference?)	MovieTutorial.Web	E:\SereneAspNetCore\MovieTutorial\9-TheCastAndCharactersTheyPlayed\AfterMoviecast\MovieTutorial\MovieTutorial.Web\Modules\Membership\Account\SignUp\AccountPage.SignUp.cs	33	Active
Error	CS0234	The type or namespace name 'Entities' does not exist in the namespace 'Serenity.Extensions' (are you missing an assembly reference?)	MovieTutorial.Web	E:\SereneAspNetCore\MovieTutorial\9-TheCastAndCharactersTheyPlayed\AfterMoviecast\MovieTutorial\MovieTutorial.Web\Modules\Administration\User\UserRepository.cs	6	Active
Error	CS0246	The type or namespace name 'IEmailSender' could not be found (are you missing a using directive or an assembly reference?)	MovieTutorial.Web	E:\SereneAspNetCore\MovieTutorial\9-TheCastAndCharactersTheyPlayed\AfterMoviecast\MovieTutorial\MovieTutorial.Web\Modules\Membership\Account\AccountPage.cs	51	Active
Error	CS0246	The type or namespace name 'IEmailSender' could not be found (are you missing a using directive or an assembly reference?)	MovieTutorial.Web	E:\SereneAspNetCore\MovieTutorial\9-TheCastAndCharactersTheyPlayed\AfterMoviecast\MovieTutorial\MovieTutorial.Web\Modules\Membership\Account\ForgotPassword\AccountPage.ForgotPassword.cs	29	Active
Error	CS0246	The type or namespace name 'EnvironmentSettings' could not be found (are you missing a using directive or an assembly reference?)	MovieTutorial.Web	E:\SereneAspNetCore\MovieTutorial\9-TheCastAndCharactersTheyPlayed\AfterMoviecast\MovieTutorial\MovieTutorial.Web\Modules\Membership\Account\ForgotPassword\AccountPage.ForgotPassword.cs	30	Active
Error	CS1739	The best overload for 'Send' does not have a parameter named 'body'	MovieTutorial.Web	E:\SereneAspNetCore\MovieTutorial\9-TheCastAndCharactersTheyPlayed\AfterMoviecast\MovieTutorial\MovieTutorial.Web\Modules\Membership\Account\SignUp\AccountPage.SignUp.cs	109	Active
![Untitled](https://user-images.githubusercontent.com/32351568/180058399-798995bd-faad-4011-86a2-0f292a5b8ec7.jpg)

**Comment by wezmag** [comment]
The `Serenity.Extensions` package should be included, when you created the project. There is no need to install.
Are you using the latest [Serene template v6.0.3.0](https://marketplace.visualstudio.com/items?itemName=VolkanCeylan.SereneSerenityApplicationTemplate)? 
Try to create a new project with the latest template, the `Serenity.Extensions` should be already installed.
